net-misc/i2pd/files/i2pd-2.6.0-fix_installed_components.patch

```diff
@@ -0,0 +1,31 @@
+diff -r -u a/build/CMakeLists.txt b/build/CMakeLists.txt
+--- a/build/CMakeLists.txt	2016-03-31 13:12:21.000000000 +0000
++++ b/build/CMakeLists.txt	2016-03-31 20:12:10.403042336 +0000
+@@ -360,10 +360,6 @@
+   endif ()
+ endif ()
+ 
+-install(FILES ../LICENSE
+-  DESTINATION .
+-  COMPONENT Runtime
+-  )
+ # Take a copy on Appveyor
+ install(FILES "C:/projects/openssl-$ENV{OPENSSL}/LICENSE"
```

Comment:
We don't want to downloading a file during the compile because it can change upstream and its not recorded in the manifest.